app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -1346,9 +1346,10 @@ def get_record_cls(record)
       record.class.base_class.name
     else
       klass = case record
-              when Host, ExtManagementSystem then record.class.base_class
-              when VmOrTemplate then              record.class.base_model
-              else                            record.class
+              when ContainerNode, ContainerGroup, Container then record.class.base_class
+              when Host, ExtManagementSystem                then record.class.base_class
```

Comment:
seems like most of the classes in the ```case get_record_cls(@record)``` clause above

I would consider getting rid of this method in a future patch and switching on true instead.
the will require checking in the main case statement what is hidden here. seems more direct to me